master/pkg/schemas/expconf/hyperparameters_config.go

```diff
@@ -23,6 +23,12 @@ type HyperparameterV0 struct {
 	RawCategoricalHyperparameter *CategoricalHyperparameterV0 `union:"type,categorical" json:"-"`
 }
 
+// Merge prevents recursive merging of hyperparameters; they are considered atomic units.
```

Comment:
is atomic the right word? maybe const after initialized?

Reply:
atomic is probably not the right word.  I'll just omit the second half.